Formula/qt.rb

```diff
@@ -213,6 +195,7 @@ def install
       -DFEATURE_pkg_config=ON
 
       -DQT_FEATURE_avx2=OFF
+      -DQT_FEATURE_webengine_proprietary_codecs=ON
       -DQT_FEATURE_webengine_system_alsa=ON
       -DQT_FEATURE_webengine_system_icu=ON
       -DQT_FEATURE_webengine_system_libevent=ON
```

Comment:
Can you take a look at issue #104008 and see if these are causing issues on macOS?